src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$transition-duration: 300ms;
+$transform-scale: scale(1.2);
```

Comment:
do you really need a variable that is only used once?

Reply:
I thought I would use it anywhere more. But I didn't :)